docs/topics/downloader-middleware.rst

```diff
@@ -876,13 +876,13 @@ Whether the Meta Refresh middleware will be enabled.
 METAREFRESH_IGNORE_TAGS
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-Default: ``[]``
+Default: ``['noscript']``
 
 Meta tags within these tags are ignored.
 
 .. versionchanged:: 2.0
    The default value of :setting:`METAREFRESH_IGNORE_TAGS` changed from
-   ``['script', 'noscript']`` to ``[]``.
+   ``[]`` to ``['noscript']``.
```

Comment:
Rather than changing this, please add a new `versionchanges` entry below this one, with the version set to `VERSION` (we’ll change that to an actual version on the next release with the change).

Reply:
The document has been updated as suggested.